CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-- Nothing yet!
+- Bump codebase to Ruby 3.1.0
```

Comment:
Maybe something like "Update official test coverage to support Ruby 3.1"? My thinking is that the code didn't change.